packages/datx-jsonapi-angular/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datx/jsonapi-angular",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
You shouldn't update this. It's done automatically by lerna during the publish process